components/next_steps_view/next_steps_tips.tsx

```diff
@@ -286,13 +335,20 @@ export default function NextStepsTips(props: { showFinalScreen: boolean; animati
                         />
                     </h2>
                 </div>
+                <CloseIcon
+                    id='closeIcon'
+                    className='close-icon'
+                    onClick={() => closeNextSteps(dispatch)}
+                />
             </header>
             <div className='NextStepsView__body'>
                 <div className='NextStepsView__nextStepsCards'>
                     <Card expanded={true}>
                         <div className='Card__body'>
-                            {// TODO: Bring back when the tour is working
-                            /* <h3>
+                            {
+
```

Comment:
ahhh eslint -- hate when it complains about not having an empty line above the comment :P